Block/Js.php

```diff
@@ -91,6 +91,12 @@ public function __construct(
         $this->httpContext = $httpContext;
     }
 
+    public function getCurrentProduct() {
+        $objectManager = \Magento\Framework\App\ObjectManager::getInstance();
+        $product = $objectManager->get('Magento\Framework\Registry')->registry('current_product');
```

Comment:
Using objectManager is not a proper magento way. @garmashdev could you please recommend how we can take the current project here?